### DISCUSSION TITLE
从 Evernote 转移到苹果自带备忘录 - Xheldon Blog

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#123] [Announcements]
https://www.xheldon.com/from-evernote-to-apple-notes.html 

 从 Evernote 转移到苹果自带备忘录

**Comment by Xheldon** [comment]
我错了...

我总共 300 多篇笔记

有一天在家里的 Mac 端新建了几篇笔记, 但是死活同步不到 iPhone 上, 尝试过各种解决方案, 依然无法通过 iCloud 同步到 iPhone, 但是 iPhone 新建的 Notes 就能同步到 Mac 上

我绝望了, 然后尝试了一个方案: 在 Mac 上先退出 Notes 的同步, 然后再打开. 就在那短短的 2s 操作后, 等我再回过头来看笔记是否同步成功的时候, 发现 Mac 端的 Notes 已经清空了, 正在从 iCloud 同步... 这确实使 Mac 端的笔记和 iPhone 端的笔记一致了... 但是 Mac 端新建的那几篇笔记丢失了
欲哭无泪, 然后就弃用了

现在继续 Notion 付费了, $48 美元还可以. 主要是排版棒, 支持嵌入代码, 同步速度快, 而且可以完整导出整个 WorkSpace 到 Markdown. 非常符合我的需求了(虽然我作死的非要上苹果全家桶)

不得不说, 苹果的软件是真的垃圾

**Comment by lisiur** [comment]
hhhhh

**Comment by Xheldon** [comment]
因为已经对 Web 构建原生 APP 的诸多限制绝望，因此现在开始使用 Craft，已经将 Notion 的全部笔记迁移过去。
Craft 有以下优点：

1. 原生技术构建，没有限制，如从外部拖拽、流畅、能第一时间用上系统级的特性
2. 支持家庭订阅共享，即一人付费，全家使用
3. 协同编辑很棒，可以创建多个空间，除了自己的私人空间（默认）外，其他的都默认是协同空间，可以邀请人协同
4. 截至目前暂不支持表格，后续会支持表格和标签，期待下！
5. 支持外部存储，即，你如果已经有了本地文件笔记，如使用 Typora 组织的笔记，Craft 可以完美识别并同步，但是外部的笔记不支持协同和分享给他人，这个需要注意。
6. Craft 没有邀请奖励机制，我这么写完全是因为喜欢，哈哈~
